setup.py

```diff
@@ -7,9 +7,8 @@
 with open("CHANGELOG.rst", encoding="UTF-8") as changelog_file:
     history = changelog_file.read()
 
-requirements = ["attrs>=18.1.0", "numpy>=1.11.0", "oop-ext>=1.1"]
+requirements = ["attrs>=18.1.0", "numpy>=1.11.0", "oop-ext>=1.1", "typing_extensions"]
```

Comment:
We do use `typing_extensions` in `barril` and were not declaring this dependency explicitly.